torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2807,6 +2807,25 @@ def sample_inputs_adaptive_avg_pool1d(op_info, device, dtype, requires_grad, **k
         # Unbatched
         yield SampleInput(make_arg(input_shape[1:]), args=(output_size,))
 
+
+def error_inputs_adaptive_avg_pool1d(opinfo, device, **kwargs):
+    make_arg = partial(make_tensor, device=device, dtype=torch.float32)
+
+    # error inputs for size overflow
+    # 0x0x3fffffffffffffff * 2 * 2 = 0xfffffffffffffffc = -4 as int64_t
+    # Tensor::numel() return int64_t, so following check that negative allocs are correctly handled
+    yield ErrorInput(SampleInput(make_arg((2, 2, 2)), kwargs={'output_size': 0x3fffffffffffffff}),
```

Comment:
fyi you can now create SampleInput's "naturally" thanks to @peterbell10, so this could create the SampleInput like:

```
SampleInput(make_arg((2, 2, 2)), output_size=0x3fffffffffffffff)
```

Reply:
Interesting! Will update the code. Thanks!